scripts/setup_olserver.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# This script is used to provision an ol-server node _before_ docker gets on it.
+
+# Which Ubuntu release are we running on?  Do not fail if /etc/os-release does not exist.
+cat /etc/os-release | grep VERSION= || true  # VERSION="20.04.1 LTS (Focal Fossa)"
+
+# CAUTION: To git clone olsystem, environment variables must be set...
+# Set $GITHUB_USERNAME or $USER will be used.
+# Set $GITHUB_TOKEN or this script will halt.
+if [[ -z ${GITHUB_TOKEN} ]]; then
+    echo "FATAL: Can not git clone olsystem" ;
+    exit 1 ;
+fi
+
+# apt list --installed
+sudo apt-get update
+sudo apt-get install -y docker.io docker-compose
+docker --version        # 19.03.8
+docker-compose version  #  1.25.0
+sudo systemctl start docker
+sudo systemctl enable docker
```

Comment:
Long term, we should replace these with the ansible-based docker install; like so:

```sh
cd /opt/olsystem
ansible-galaxy install -r playbooks/requirements.yml
ansible-playbook --become --inventory="playbooks/inventory" --limit="ol-solr0" "playbooks/site.yml"
```

(Note this needs to be done _after_ we `git clone`). But not sure what the best `limit` is, since `ol-solr0` has special ferm rules, too.